Formula/modsman.rb

```diff
@@ -0,0 +1,31 @@
+class Modsman < Formula
+  desc "Minecraft mod manager for the command-line"
+  homepage "https://gitlab.com/sargunv-mc-mods/modsman"
+
+  # for new versions, update the url's build number and artifact name, the sha256, and the version
+  url "https://dev.azure.com/sargunvohra/modsman/_apis/build/builds/37/artifacts?artifactName=modsman-cli-0.20.1&api-version=5.0&%24format=zip"
```

Comment:
The gitlab page shows no releases, and we don't want to download from another (untrusted) location.

Reply:
The [Gitlab page](https://gitlab.com/sargunv-mc-mods/modsman/blob/master/README.md) now links to the releases on Azure. If you compare the URL from the build details page, you'll notice it's the exact same download link. Additionally, he sha256 should ensure the file is always what we expect. This project builds on Azure instead of Gitlab CI because Azure supports all thee (macos/linux/windows) platforms in its CI infra.